Assignment1

joe83830 opened the pull request:
jun-hsiang-liao jl3744

joe83830 committed:
Trial

amfleming reviewed (changes requested):
You need to pull from upstream to avoid conflicts

And don't delete the Alpha-order comment

joe83830 committed:
Trial1

joe83830 committed:
In Alphabetical Order

joe83830 commented:
I modified the files

amfleming commented on README.md:
```diff
@@ -47,6 +47,7 @@ People
 
 
 #### Students
+
```
Why new line?

amfleming commented on README.md:
```diff
@@ -58,6 +59,7 @@ People
 * [Rom Cohen](people/rom-cohen.md) - rc783 - [https://cs5356-rc783.s3-website.us-east-2.amazonaws.com/grading.html](https://cs5356-rc783.s3-website.us-east-2.amazonaws.com/grading.html)
 * [Marika Cusick](people/marika-cusick.md) - mmc265 - [https://s3.us-east-2.amazonaws.com/marikacusickbucket/graded.html](https://s3.us-east-2.amazonaws.com/marikacusickbucket/graded.html)
 * [Jonathan Cutler](people/jonathan-cutler.md) - jlc553 - [https://s3.us-east-2.amazonaws.com/cs5356-jlc553/grading.html](https://s3.us-east-2.amazonaws.com/cs5356-jlc553/grading.html)
+* [jun-hsiang-liao](people/jun-hsiang-liao.md) - jl3744 - [http://jl3744.xyz.s3-website.us-east-2.am    azonaws.com](http://jl3744.xyz.s3-website.us-east-2.amazonaws.com/grading.html)
```
Shouldn't this start with `[Jun Liao]` ?

amfleming commented on README.md:
```diff
@@ -97,6 +99,7 @@ People
 * [Elena Zhizhimontova](people/elena-zhizhimontova.md) - ez256 - [https://s3-us-west-2.amazonaws.com/zhizhimontova.com/grading.html](https://s3-us-west-2.amazonaws.com/zhizhimontova.com/grading.html)
 * [Yuxiang Zhu](people/yuxiang-zhu.md) - yz2329 - [https://s3.amazonaws.com/cs5356-yz2329/grading.html](https://s3.amazonaws.com/cs5356-yz2329/grading.html)
 
+>>>>>>> 67600b5a447043068f7b780c576b13d67e0341f4
```
What's this?

amfleming commented on people/jun-hsiang-liao.md:
```diff
@@ -0,0 +1,7 @@
+Adam Fleming
```
Shouldn't this say `Jun Liao`?

amfleming reviewed (changes requested):
Yikes

joe83830 committed:
Name fixed

joe83830 commented:
Sry I removed the weird thing on line 102.
I also Capitalized my name and removed the dashes in between.
But actually my name is Jun-Hsiang Liao, it's Mandarin so there are two words in my first name and this is how it appears on my passport.
I changed it to Jun Hsiang Liao.
Or should I combine it into sth like JunHsiang Liao?
Sorry for the trouble :P

amfleming commented:
Hi! You can spell your name however you want

I'm certain you don't spell it `Adam Fleming` however

joe83830 committed:
Name_Fixed_2

joe83830 committed:
Name_fixed_3

joe83830 commented:
I think I corrected the format.
This is my first time forking things on GitHub, hope I finally got everything working, thanks!

amfleming commented on people/junhsiang-liao.md:
```diff
@@ -0,0 +1,7 @@
+Adam Fleming
```
Still says `Adam Fleming` 😬

amfleming reviewed (changes requested):
Hi! Still have an accidental new line 

And people/junhsiang-liao.md still stays 'Adam Fleming'

joe83830 committed:
Ahhhhh

joe83830 commented:
Noooooo I'm terribly sorry sir I finally understood what you meant!

joe83830 committed:
New Line
